src/app/components/contentItem/contentTile.component.scss

```diff
@@ -83,20 +88,26 @@
 }
 
 .content-tile-title {
-    @extend %rowCentered;
+    @extend %columnLeft;
     @extend %marginZero;
-    @extend %paletteDarkContrast;
+    color: $color_text-dark;
     font-size: 0.9rem;
-    height: 90px;
-    padding: 15px;
+    height: 100%;
+    padding: 0 0 15px 0;
```

Comment:
wouldn't padding-bottom: 15px be enough?

Reply:
changed to padding-bottom